src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -7,10 +7,12 @@
     <Version>1.11.1-prerelease</Version>
     <Title>Datadog APM - ClrProfiler</Title>
     <Description>OBSOLETE. This package is deprecated and exists only for backwards compatibility.
-      Users who need manual instrumentation should reference the "Datadog.Trace" package.
-      Users who need ASP.NET instrumentation should reference the "Datadog.Trace.AspNet" package.
+Users who need automatic instrumentation on .NET Core do not need a NuGet package anymore.
+Users who need automatic instrumentation on legacy ASP.NET instrumentation should reference the "Datadog.Trace.AspNet" package.
```

Comment:
"instrumentation on legacy ASP.NET instrumentation" should be reworded, but I don't have any suggestions

Reply:
Oops. I dropped the second "instrumentation" in fc181321087f3cba2257307e779c66b576f363c1 so it reads:

> Users who need automatic instrumentation on legacy ASP.NET should reference the "Datadog.Trace.AspNet" package.